en_US/mqtt/auto_subscribe/auto_subscribe.md

```diff
@@ -0,0 +1,93 @@
+# MQTT
```

Comment:
```suggestion

```

Reply:
the heading-1 is used to generate navigation menu.
should match the title of the doc.